src/main/java/org/candlepin/model/Consumer.java

```diff
@@ -528,22 +508,46 @@ public Consumer setOwner(Owner owner) {
         return this;
     }
 
-    @Override
-    public String toString() {
-        return String.format("Consumer [id: %s, uuid: %s, name: %s]",
-            this.getId(), this.getUuid(), this.getName());
+    /**
+     * Check if the consumers facts have changed and contains the cloud profile facts.
+     * It returns true if incoming facts are cloud profile facts
+     *
+     * @param incomingFacts incoming facts
+     * @return a boolean
```

Comment:
We could add a better description of the returned value here

Reply:
Fixed.